OpenSim/Tools/Test/testVisualization.cpp

```diff
@@ -98,21 +99,27 @@ class DecorativeGeometryImplementationText : public SimTK::DecorativeGeometryImp
     std::string getAsString() {
         return printout.str();
     }
+    void setPrintTransforms(bool toPrint) { printTransform = toPrint;}
 private:
     std::stringstream printout;
+    bool printTransform; // Flag to indicate whether or not to output transform as String
```

Comment:
You can get rid of the lines like `dgiTextFromStandard.setPrintTransforms(false);` by changing this member declaration to the following:

```
bool printTransform = false;
```

(C++11 non-static member initializer).